src/main/java/co/omise/Configurer.java

```diff
@@ -16,7 +15,7 @@
     private final Config config;
 
     Configurer(Config config) {
-        Preconditions.checkNotNull(config);
+        ConditionUtility.notNull(config);
```

Comment:
Not sure, why not use `@NotNull`?

Reply:
That stops you from directly passing null as a param but doesn't protect the method if an error somewhere else in the code causes a null to be passed in.